client/bio-commandline/src/main/kotlin/uk/ac/ebi/biostd/client/cli/BioStudiesCommandLine.kt

```diff
@@ -36,13 +37,14 @@ class BioStudiesCommandLine : CliktCommand(name = "PTSubmit") {
             }
 
             val client = getClient(server, user, password)
-            val submission = client.submitSingle(input!!.readText(), SubmissionFormat.valueOf(format), files).body!!
+            val submission =
+                client.submitSingle(input!!.readText(), SubmissionFormat.valueOf(format.toUpperCase()), files).body!!
 
             echo("SUCCESS: Submission with AccNo ${submission.accNo} was submitted")
         } catch (exception: Exception) {
             when (exception) {
                 is ResourceAccessException -> throw PrintMessage(exception.cause?.message ?: FILES_NOT_FOUND_ERROR_MSG)
-                is RestClientResponseException -> throw PrintMessage(toJson(exception))
+                is RestClientResponseException -> throw PrintMessage(formatException(exception))
```

Comment:
I think we should start doing this at the client level, catch any error and return proper JSON from there. Yet another TODO